pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -390,6 +395,17 @@ var supportedReinvocationPolicies = sets.NewString(
 	string(admissionregistration.IfNeededReinvocationPolicy),
 )
 
+var supportedValidationPolicyReason = sets.NewString(
+	string(metav1.StatusReasonUnauthorized),
+	string(metav1.StatusReasonForbidden),
+	string(metav1.StatusReasonNotFound),
+	string(metav1.StatusReasonInvalid),
+	string(metav1.StatusReasonTooManyRequests),
```

Comment:
since the evaluation is limited to a single request, I don't think a 429 makes sense for this to return, so drop this one